pkg/services/services.go

```diff
@@ -56,3 +59,32 @@ func hybridOverlayConfiguration(vxlanPort string, debug bool) servicescm.Service
 		Priority:                     1,
 	}
 }
+
+func kubeProxyConfiguration() servicescm.Service {
```

Comment:
If kube-proxy has debug levels, then this should follow the same treatment that `hybridOverlayConfiguration()` got i.e. take `debug` as input,